src/Client/ProcessControl/Requests/TerminateProcessInstanceRequest.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace AtlasEngine.ProcessControl.Requests
+{
+  using System;
+  using System.Net.Http;
+  using AtlasEngine.ApiClient;
+
+  public sealed class TerminateProcessInstanceRequest : BaseRequest<NoPayload>
+  {
+    public TerminateProcessInstanceRequest(string processInstanceId)
+    {
+      this.ProcessInstanceId = processInstanceId;
+    }
+
+    public string ProcessInstanceId { get; }
+
+    public override void EnsureIsValid()
+    {
+      if (string.IsNullOrWhiteSpace(this.ProcessInstanceId))
+      {
+        throw new ArgumentException("No process instance specified.", nameof(this.ProcessInstanceId));
+      }
+    }
+
+    public override (HttpMethod, Uri, object) BuildRequestData()
+    {
+      var uri = new Uri($"/{Constants.ManagementBaseUri}/process_instance/{this.ProcessInstanceId}/terminate", UriKind.Relative);
```

Comment:
Die Route müsste `process_instances` sein

```suggestion
      var uri = new Uri($"/{Constants.ManagementBaseUri}/process_instances/{this.ProcessInstanceId}/terminate", UriKind.Relative);
```

Zumindest in der AE. Ich schaue mal eben wie das in der PE ist.